db/migrate/20191206142959_create_products.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProducts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :products do |t|
+      t.string :name
+      t.float :price
```

Comment:
Hier fehlen noch definitiv die Indizes auf Product `Name` und `Price`. Du willst die doch in der Datenbank immer nach diesen Sortieren und dann auch suchen. Ohne Index ist das nicht schnell!